semcore/tag/src/index.d.ts

```diff
@@ -83,8 +83,21 @@ export type TagTextProps = BoxProps & KeyboardFocusProps & {};
 declare const Tag: Intergalactic.Component<'div', TagProps, TagContext> & {
   Text: Intergalactic.Component<'div', TagTextProps>;
   Addon: Intergalactic.Component<'div', TagAddonProps>;
+  /**
+   * @deprecated For Tags with close button, you should use
```

Comment:
Let's just keep a link to the full example here

Reply:
added link to the example in documentation